utils/storybook/CanvasProviderDecorator.tsx

```diff
@@ -3,13 +3,16 @@ import {
   CanvasProvider,
   defaultCanvasTheme,
   createCanvasTheme,
+  CanvasTheme,
 } from '@workday/canvas-kit-labs-react-core';
 import {object} from '@storybook/addon-knobs';
 
 const label = 'theme';
 
-export default (storyFn: () => React.ReactNode) => (
-  <CanvasProvider theme={createCanvasTheme(object(label, defaultCanvasTheme))}>
+export default (storyFn: () => React.ReactNode, customTheme?: CanvasTheme) => (
+  <CanvasProvider
+    theme={customTheme ? customTheme : createCanvasTheme(object(label, defaultCanvasTheme))}
```

Comment:
also open to this approach. This allows for the custom theme to override the default theme we have for each component. I added an extra param so that we can pass whatever custom theme object we'd like